packages/compass-saved-aggregations-queries/src/components/saved-item-card.tsx

```diff
@@ -26,6 +26,7 @@ export type SavedItemCardProps = Pick<
   'id' | 'type' | 'name' | 'database' | 'collection' | 'lastModified'
 > & {
   onAction(id: string, actionName: Action): void;
+  dataTestId?: string;
```

Comment:
Why are we not using `data-testid` here? Why introduce a new name for the variable on top of existing one? On top of that card already accepts all possible html props, why do we need this here?